src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 logging.config=classpath:logback-base.xml
-
 pn.env.runtime=PROD
+pn.selfcare.baseurl=https://selfcare.pagopa.it
```

Comment:
I don't think that it goes here. We should add the properties into microservice-dev-cfg.json.
Check the other ms to understand how it should be done

N.B. check how to add the configuration for others environments (test, uat, hotfix and prod)